network/common/src/main/java/org/apache/spark/network/util/JavaUtils.java

```diff
@@ -25,13 +32,6 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
-import com.google.common.base.Charsets;
-import com.google.common.base.Preconditions;
-import com.google.common.collect.ImmutableMap;
-import io.netty.buffer.Unpooled;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
```

Comment:
this was in the right place before. Can you move it back? In general the ordering should be
java -> scala -> third party -> spark imports